starlette_admin/translations/de/LC_MESSAGES/admin.po

```diff
@@ -0,0 +1,191 @@
+# German translations for starlette-admin.
+# Copyright (C) 2024 ORGANIZATION
+# This file is distributed under the same license as the starlette-admin
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: starlette-admin VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2024-03-07 16:31+0000\n"
+"PO-Revision-Date: 2024-03-07 16:50+0000\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: de\n"
+"Language-Team: de <opensource@bakdata.com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.13.0\n"
+
+#: starlette_admin/actions.py:11 starlette_admin/actions.py:107
+#: starlette_admin/templates/modals/actions.html:12
+msgid "Yes, Proceed"
+msgstr "Ja, fortfahren"
+
+#: starlette_admin/auth.py:31
+msgid "Administrator"
+msgstr "Administrator"
+
+#: starlette_admin/base.py:44 starlette_admin/contrib/odmantic/admin.py:17
+#: starlette_admin/contrib/sqla/admin.py:28
+#: starlette_admin/templates/create.html:7
+#: starlette_admin/templates/detail.html:8
+#: starlette_admin/templates/edit.html:7 starlette_admin/templates/list.html:7
+msgid "Admin"
+msgstr "Admin"
+
+#: starlette_admin/base.py:254
+#, python-format
+msgid "Model with identity %(identity)s not found"
+msgstr "Modell mit Identität %(identity)s wurde nicht gefunden"
+
+#: starlette_admin/templates/forms/_delete.html:8
+#: starlette_admin/templates/modals/delete.html:25 starlette_admin/views.py:461
+#: starlette_admin/views.py:498
+msgid "Delete"
+msgstr "Löschen"
+
+#: starlette_admin/views.py:462
+msgid "Are you sure you want to delete selected items?"
+msgstr "Sind Sie sicher, dass Sie die ausgewählten Elemente löschen möchten?"
+
+#: starlette_admin/views.py:463
+msgid "Yes, delete all"
+msgstr "Ja, alles löschen"
+
+#: starlette_admin/views.py:468 starlette_admin/views.py:507
+#, python-format
+msgid "Item was successfully deleted"
+msgid_plural "%(count)d items were successfully deleted"
+msgstr[0] "Element wurde erfolgreich gelöscht"
+msgstr[1] "%(count)d Elemente wurden erfolgreich gelöscht"
+
+#: starlette_admin/views.py:476
+msgid "View"
+msgstr "Ansehen"
+
+#: starlette_admin/templates/edit.html:12 starlette_admin/views.py:488
+msgid "Edit"
+msgstr "Bearbeiten"
+
+#: starlette_admin/templates/modals/delete.html:15 starlette_admin/views.py:499
+msgid "Are you sure you want to delete this item?"
+msgstr "Sind Sie sicher, dass Sie dieses Element löschen möchten?"
+
+#: starlette_admin/views.py:818
+msgid "All"
+msgstr "Alle"
+
+#: starlette_admin/templates/actions.html:6
+msgid "With selected"
+msgstr "Mit ausgewählten"
+
+#: starlette_admin/templates/create.html:12
+msgid "Create"
+msgstr "Erstellen"
+
+#: starlette_admin/templates/create.html:25
+#: starlette_admin/templates/list.html:34
+#, python-format
+msgid "New %(name)s"
+msgstr "%(name)s erstellen"
```

Comment:
this is not a 1:1 translation because the grammar is a little different in German. It would need to be "Neuer Post" but "Neues Modell" for example, therefore I opted for a different wording that works for all cases